autoscheduler/frontend/src/components/Schedule/Schedule.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { RouteComponentProps } from '@reach/router';
+import * as styles from './Schedule.css';
+import Meeting from '../../types/Meeting';
+import MeetingCard from '../MeetingCard/MeetingCard';
+
+interface ScheduleProps extends RouteComponentProps {
+  schedule: Meeting[];
+}
+
+const Schedule: React.FC<ScheduleProps> = ({ schedule }) => {
+  // these must be unique because of how they're used below
+  const DAYS_OF_WEEK = ['M', 'T', 'W', 'R', 'F'];
+  const FIRST_HOUR = 8;
+  const LAST_HOUR = 21;
+
+  // helper function for formatting
+  function formatHours(hours: number): number {
+    return ((hours - 1) % 12) + 1;
+  }
+
+  /* values computed from props */
+  const uniqueSections = [...new Set([...schedule.map((mtg: Meeting) => mtg.section.id)])];
```

Comment:
```suggestion
  const uniqueSections = [...new Set(schedule.map((mtg: Meeting) => mtg.section.id))];
```
Not necessary to spread `map()`